packages/ckeditor5-table/src/plaintableoutput.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module table/plaintableoutput
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+import Table from './table';
+
+/**
+ * The plain table output feature.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class PlainTableOutput extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'PlainTableOutput';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ Table ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+
+		// Override default table data downcast converter.
+		editor.conversion.for( 'dataDowncast' ).elementToStructure( {
+			model: 'table',
+			view: downcastTableElement,
+			converterPriority: 'high'
+		} );
+
+		// Make sure <caption> is always downcasted to <caption> in the data pipeline.
+		editor.conversion.for( 'dataDowncast' ).elementToElement( {
+			model: 'caption',
+			view: 'caption',
+			converterPriority: 'high'
+		} );
```

Comment:
Does it work ok with images with captions? AFAIK image caption should not be affected by this change, but this converter looks like it would take over it.